datus-clickhouse/tests/integration/conftest.py

```
@@ -26,9 +26,22 @@ def connector(config: ClickHouseConfig) -> Generator[ClickHouseConnector, None,
+            database=None,
+        )
+        init_conn = ClickHouseConnector(init_config)
+        if not init_conn.test_connection():
             pytest.skip("Database connection test failed")
+        if config.database:
+            init_conn.execute_ddl(f"CREATE DATABASE IF NOT EXISTS `{config.database}`")
+        init_conn.close()
+
+        conn = ClickHouseConnector(config)
         yield conn
     except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)